static/js/src/static-forms.js

```diff
@@ -21,10 +21,22 @@
   });
 })();
 
```

Comment:
I would first try to find a form on the page with something like:
`const form = document.querySelector("form[action='/marketo/submit']");`
Check if this exists and if it doesy ou can then find the button within the form (`form.querySelector("button[type='submit']"`) and attach the event listener that runs the rest of the code.